make/test/BuildTestSetupAOT.gmk

```diff
@@ -53,9 +54,9 @@ TARGETS += $(BUILD_SETUP_AOT)
 
 # Copy to hotspot jtreg test image
 $(eval $(call SetupCopyFiles, COPY_SETUP_AOT, \
-    SRC := $(SETUP_AOT_SUPPORT)/classes, \
+    SRC := $(SETUP_AOT_SUPPORT), \
```

Comment:
With just one file being copied, the SRC argument can be omitted. It will default to the directory of the first file in FILES.